examples/2d/elixir_advection_callbacks.jl

```diff
@@ -0,0 +1,154 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+
+# define new structs inside a module to allow re-evaluating the file
+module TrixiExtensionExample
+
+using Trixi
+using DiffEqCallbacks: DiscreteCallback, u_modified!
+
+
+struct ExampleLimiter
+  times::Vector{Float64}
+  min_values::Vector{Float64}
+  max_values::Vector{Float64}
+
+  # You can optionally define an inner constructor to set up some required stuff.
+  function ExampleLimiter()
+    new(Float64[], Float64[], Float64[])
+  end
+end
+
+# This method is called when the `ExampleLimiter` is used as `stage_limiter!`
+# which gets called after every RK stage. There is no specific initialization
+# method for such `stage_limiter!`s in OrdinaryDiffEq.jl.
+function (example_limiter::ExampleLimiter)(u_ode::AbstractVector, _, semi, t)
+
+  min_val, max_val = extrema(u_ode)
+  push!(example_limiter.times, t)
+  push!(example_limiter.min_values, min_val)
+  push!(example_limiter.max_values, max_val)
+
+  return nothing
+end
+
+
+struct ExampleCallback
```

Comment:
```suggestion
# This is an example implementation for a simple callback (i.e., a callable that is potentially executed after each Runge-Kutta step), which records some values each time it is called. Its sole purpose here is to showcase how to implement a callback for Trixi.
struct ExampleCallback
```

Reply:
Added in 8f2fe48da94f56682ccfd55b2a8bdec0e6052337.